recipes/assertpy/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: assertpy
+  version: !!str 0.12
+
+source:
+  fn: assertpy-0.12.tar.gz
+  url: https://pypi.python.org/packages/54/66/1ed435aa2f51e39d73ff4c59dc9bc569278fcaf9586599891b146a8f9e5a/assertpy-0.12.tar.gz#md5=e0d382ca6e3788e564f0ec6c24162c37
```

Comment:
You can use the canonical url shown here:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

once you use the jinja templating variables